docs/event-bus/08-01-configure-default-channelprovisioner.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Subscription
+type: Configure default ClusterChannelProvisioner
+---
+
+Kyma comes with NATSS as its default ClusterChannelProvisioner(See [default-channel-webhook](../../resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml) ConfigMap). Other than NATSS, one can use any default channel provisioner of one's choice. In order to add a in-memory-channel provisioner follow this [guide](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel).
+
+> **Note**: Before installing this provisioner, add the annotation below to get rid of the istio-sidecar [here](https://github.com/knative/eventing/blob/master/config/provisioners/in-memory-channel/in-memory-channel.yaml#L107) for the in-memory-channel-controller.
+```yaml
```

Comment:
we can provide a few more lines for the yaml to make it recognizable.

Reply:
added